storage/post.go

```diff
@@ -284,19 +285,13 @@ func (m *Miner) computePost(ppe uint64) func(ctx context.Context, ts *types.TipS
 			ppe: ppe,
 			ts:  ts,
 		}).doPost(ctx)
+
+		m.scheduleNextPost(ppe + build.ProvingPeriodDuration)
```

Comment:
Ah, so we make sure to schedule the next one even if this one failed?

Reply:
Yep